### PR TITLE
Move event stream messages to EventStream.*, #27211

### PR DESCRIPTION
**Pull request by patriknw**
* for better prefix usage, e.g. EventStream.Publish
* and consistency with Receptionist

Refs #27211

some other cleanup at the same time

### CODE REVIEW

File: akka-actor-typed-tests/src/test/java/akka/actor/typed/eventstream/EventStreamTest.java [hunk 1]
```diff
@@ -12,7 +12,6 @@
   static class SomeClass {}
 
   public static void compileOnlyTest(ActorSystem<?> actorSystem, ActorRef<SomeClass> actorRef) {
-    actorSystem.eventStream().tell(Subscribe.of(SomeClass.class, actorRef));
-    actorSystem.eventStream().tell(new Subscribe<>(SomeClass.class, actorRef));
+    actorSystem.eventStream().tell(new EventStream.Subscribe<>(SomeClass.class, actorRef));
```
Comment (johanandren) [inline akka-actor-typed-tests/src/test/java/akka/actor/typed/eventstream/EventStreamTest.java]: No coverage of Subscribe anymore?

**Review by johanandren — APPROVED** [review]

File: akka-actor-typed-tests/src/test/java/akka/actor/typed/eventstream/EventStreamTest.java (hunk 1 above)
Comment (patriknw) [reply akka-actor-typed-tests/src/test/java/akka/actor/typed/eventstream/EventStreamTest.java]: `Subscribe.of` was removed, `new EventStream.Subscribe, is still covered here

**Review by chbatey — APPROVED** [review]
LGTM with a note in the typed migration guide

**Comment by chbatey** [comment]
Or was this only added in 2.6?

**Comment by johanandren** [comment]
Yup, #21525 2.6.0-M2

**Pull request merged**